stilistische Änderungsvorschläge L499 - 524

jahfaby opened the pull request.

jahfaby committed:
stilistische Änderungsvorschläge L499 - 524

TQueV commented on words/words.md:
```diff
@@ -498,30 +498,30 @@ Trotzdem kennen wir weiterhin nicht den *spezifischen* Einfluss von Masken auf d
 
 
-Pandemien scheinen einem Pokerspiel zu gleichen: **Wette nur, wenn du dir 95% sicher bist mit deinem ganzen Einsatz!** Wie aus einem aktuellen Artikel[^precautionary] über Masken im British Medical Journal hervorging, *müssen* unsere Kosten/Nutzen-Analysen mit einem Unsicherheitsfaktor behaftet sein. 
+Pandemien sind wie Poker: **Wette nur, wenn du dir zu 95% sicher bist und du sonst alles verlieren wirst.** Wie aus einem aktuellen Artikel[^precautionary] über Masken im British Medical Journal hervorgeht, *müssen* unsere Kosten/Nutzen-Analysen mit einem Unsicherheitsfaktor berechnet werden. In etwa so:
```
Top!

TQueV commented on words/words.md:
```diff
@@ -498,30 +498,30 @@ Trotzdem kennen wir weiterhin nicht den *spezifischen* Einfluss von Masken auf d
 [^precautionary]: “It is time to apply the precautionary principle” [Trisha Greenhalgh et al \[PDF\]](https://www.bmj.com/content/bmj/369/bmj.m1435.full.pdf)
 
-Kosten: Alltagsmasken sind günstig (und etwa ~2/3 so effektiv wie chirurgische Masken[^homemade]) sind). 
+Kosten: Wenn Alltagsmasken (die etwa ~2/3 so effektiv wie chirurgische Masken[^homemade]) sind), dann ist das superbillig. Wenn chirurgische Masken, dann wird das teurer, aber immer noch ziemlich billig.
```
Vorschlag: "Wenn es sich um chirurgische Masken handelt, wird es teurer, aber immer noch ziemlich billig."

jahfaby replied:
Ja, können wir gerne machen. Ich habe nur keine Ahnung, wie ich das hier mache. :/ :)

herrhundt replied:
erledigt

herrhundt committed:
Merge branch 'master' into jahfaby-patch-1

herrhundt committed:
Update words.md - Korrektur Alltagsmasken

(...) Wenn es sich um chirurgische Masken handelt, wird es teurer, aber immer noch ziemlich billig. (...)
